src/global_commands.c

```diff
@@ -738,7 +738,7 @@ void cmd_log(WINDOW *window, ToxWindow *self, Toxic *toxic, int argc, char (*arg
 
     if (!strcmp(swch, "1") || !strcmp(swch, "on")) {
         if (log_enable(log) == 0) {
-            char e_msg[MAX_STR_SIZE];
+            char e_msg[MAX_STR_SIZE + 20];
```

Comment:
Might be nice to create a define for this since it's not obvious what the `+ 20` is based on?

Reply:
Turns out we don't need the buffer at all